core/mm4Method.js

```diff
@@ -255,7 +255,7 @@ CLASS({
     function maybeCall(X, that) { /* Executes this action if $$DOC{ref:'.isEnabled'} is allows it. */
       if ( this.isAvailable.call(that, this) && this.isEnabled.call(that, this) ) {
         this.code.call(that, X, this);
-        that.publish(['action', this.name], this);
+        if (that && that.publish) that.publish(['action', this.name], this);
```

Comment:
When called from `js/foam/u2/md/ActionButton.js` line 132, it passes `this.data` as `that`. For a view with no `data` that does `this.add(this.MY_ACTION)` for an action of its own, `this.data` is blank. Seems like a legit case to me.

Or maybe the `data` of the `ActionButton` should always be the instance that owns the `Action`?